server/database/schema.sql

```diff
@@ -48,16 +48,16 @@ CREATE TABLE menu (
 CREATE TABLE recipe (
     id INT PRIMARY KEY AUTO_INCREMENT NOT NULL,
     name VARCHAR(255) NOT NULL,
-    number_of_people INT,
+    number_of_people INT UNSIGNED NOT NULL,
     description TEXT NOT NULL,
     image TEXT NOT NULL,
-    date DATE NOT NULL,
-    is_favorite BOOLEAN,
-    vote INT,
-    set_up_time INT,
-    is_validated BOOLEAN,
-    user_id INT NOT NULL,
-    badge_id INT NOT NULL,
+    date DATE DEFAULT "1900-01-01",
+    is_favorite BOOLEAN DEFAULT false,
+    vote INT UNSIGNED DEFAULT 0,
+    set_up_time INT UNSIGNED NOT NULL,
+    is_validated BOOLEAN DEFAULT false,
+    user_id INT DEFAULT 1,
+    badge_id INT DEFAULT 1,
```

Comment:
No, no and no ! 
Never put a default foreign key value !

Reply:
I updated it